src/components/navs/AppNav/AppNavActivityBtn/AppNavActivityBtn.vue

```diff
@@ -70,31 +79,65 @@ export default defineComponent({
   },
 
   setup() {
-    // COMPOSABLES
+    /**
+     * COMPOSABLES
+     */
     const { upToLargeBreakpoint } = useBreakpoints();
-    const { isLoadingProfile, profile, account } = useWeb3();
+    const { isLoadingProfile, profile, account, getSigner } = useWeb3();
+    const { t } = useI18n();
+
     const {
       transactions,
       pendingTransactions,
+      finalizedTransactions,
       getExplorerLink,
       clearAllTransactions,
-      isSuccessfulTransaction
+      isSuccessfulTransaction,
+      updateTransaction,
+      isPendingTransactionStatus
     } = useTransactions();
 
-    // COMPUTED
-    const unconfirmedTransactions = computed(() =>
-      transactions.value.filter(({ status }) => status !== 'confirmed')
-    );
+    const { addNotification } = useNotifications();
+
+    /**
+     * METHODS
+     */
+    async function cancelOrder(orderId: string) {
+      try {
+        const { signature, signingScheme } = await signOrderCancellation(
+          orderId,
+          getSigner()
+        );
+
+        await gnosisOperator.sendSignedOrderCancellation({
+          cancellation: {
+            orderUid: orderId,
+            signature,
+            signingScheme
+          },
+          owner: account.value
+        });
 
-    const confirmedTransactions = computed(() =>
-      transactions.value.filter(({ status }) => status === 'confirmed')
-    );
+        updateTransaction(orderId, 'order', {
+          status: 'cancelling'
+        });
+      } catch (e) {
+        console.log(e);
+        addNotification({
+          type: 'error',
+          title: t('errorCancellingOrder'),
+          message: e.message
```

Comment:
pretty handy to throw this kind of errors when things go wrong... in this instance an attempt to close an order was not successful (perhaps it was already filled)